lib/angular/service.js

```diff
@@ -42,20 +45,16 @@ function wrapManager($q, $timeout, manager) {
   return wrappedManager;
 }
 
-angular.module('wfm.result.sync', [require('fh-wfm-sync')])
-.factory('resultSync', function($q, $timeout, syncService) {
-  syncService.init($fh, config.syncOptions);
+angular.module('wfm.result.sync', [])
+.factory('resultSync', function($q, $timeout, mediator) {
   var resultSync = {};
   resultSync.createManager = function(queryParams) {
     if (resultSync.manager) {
       return $q.when(resultSync.manager);
     } else {
-      return resultSync.managerPromise = syncService.manage(config.datasetId, null, queryParams)
-        .then(function(manager) {
-          resultSync.manager = wrapManager($q, $timeout, manager);
-          console.log('Sync is managing dataset:', config.datasetId, 'with filter: ', queryParams);
-          return resultSync.manager;
-        });
+      resultSync.manager = wrapManager($q, $timeout, resultClient(mediator));
+      console.log('Sync is managing dataset:', config.datasetId, 'with filter: ', queryParams);
```

Comment:
@wtrocki I think id does not matter. We can remove it in one run. In this case focusing only on decoupling